libafl/src/stages/calibrate.rs

```diff
@@ -92,6 +92,8 @@ where
     E::State: HasCorpus + HasMetadata + HasNamedMetadata + HasExecutions,
     Z: Evaluator<E, EM, State = E::State>,
 {
+    type Status = (); // TODO stage may be resumed, but how?
```

Comment:
Not sure if `Status` is very obvious as a name? `Progress` or `ResumeState`? (sorry I'm just nitpicking on words here :D )